x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/state_management/stream_enrichment_state_machine/stream_enrichment_state_machine.ts

```diff
@@ -215,43 +194,36 @@ export const streamEnrichmentMachine = setup({
   id: 'enrichStream',
-  context: ({ input }) => ({
+  context: ({ input, spawn }) => ({
     definition: input.definition,
     dataSourcesRefs: [],
     grokCollection: new GrokCollection(),
     initialProcessorsRefs: [],
     processorsRefs: [],
     urlState: defaultEnrichmentUrlState,
+    simulatorRef: spawn('simulationMachine', {
+      id: 'simulator',
+      input: {
+        processors: [],
+        streamName: input.definition.stream.name,
+      },
+    }),
   }),
-  initial: 'initializingStream',
+  initial: 'initializingFromUrl',
   states: {
-    initializingStream: {
```

Comment:
Good call pulling that out of the state machine - it was just noise basically and it's cleaner to simply not render the component